File: frontend/src/app/models/create-employee-request.ts
export class CreateEmployeeRequest {
  username: string;
  password: string;

  firstName: string;
  lastName: string;
  telephone: string;
  address: string;
  city: string;
  state: string;
  zip: number;

  ssn: number;
  startDate: Date;
  hourlyRate: number;
}

File: backend/src/main/java/flightbook/entity/employee/CreateEmployeeRequest.java
package flightbook.entity.employee;

import lombok.Data;

import java.util.Date;

@Data
public class CreateEmployeeRequest {
	private String username;
	private String password;

	private String firstName;
	private String lastName;
	private String telephone;
	private String address;
	private String city;
	private String state;
	private int zip;

	private int SSN;
	private Date startDate;
	private double hourlyRate;
}
